app/scripts/views/mixins/settings-panel-mixin.js

```diff
@@ -53,4 +53,24 @@ define(function (require, exports, module) {
       self.closePanel();
     }
   };
+  if (window.history && window.history.pushState) {
```

Comment:
This solution starts the behavior on application start instead of via an initialization function. Behavior added at startup in this manner is difficult to unit test in isolation and is prone to cause unwanted side effects.

I'm trying to think of where this functionality should live, settings-panel-mixin is added to 11 views, so adding the DOM event handlers in this module's initialization function would likely cause problems too.